app/models/user.rb

```diff
@@ -79,4 +80,8 @@ def self.non_admins
   def self.without_questionnaire
     non_admins.left_outer_joins(:questionnaire).where(questionnaires: { id: nil })
   end
+
+  def save_audit_information
+    @deleted_user = ::DeletedUser.new(user_id: id, email: email).save
```

Comment:
Idk if it's best to keep PII (email address) around if the user deletes their account. Maybe consider scoping this to only store a deleted user if that user was an admin?